### ISSUE TITLE
Whitepaper Reference is an invalid URL

### ISSUE THREAD

**Issue by naivary** [cncf/tag-app-delivery#544]
In the following [whitepaper](https://github.com/cncf/tag-app-delivery/blob/163962c4b1cd70d085107fc579e3e04c2e14d59c/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md) is an invalid URL. The invalid URL is the reference number six with the following URL: https://github.com/cncf/sig-security/blob/master/security-whitepaper/cloud-native-security-whitepaper.md.

**Comment by abangser** [comment]
Great shout @naivary , I believe they have put out a v2 and the URL is: https://github.com/cncf/tag-security/blob/main/security-whitepaper/v2/cloud-native-security-whitepaper.md

Is this something you are comfortable opening a PR for? The only thing to keep in mind is that we will need to put it in each of the different language versions.

**Comment by naivary** [comment]
I'm going to have a look at it tommorow 😄

**Comment by naivary** [comment]
Can you maybe indicate me where the different language versions are?

**Comment by abangser** [comment]
Absolutely. The website content is all found here: https://github.com/cncf/tag-app-delivery/tree/main/website/content

Within there is each language we support and each should have the same structure. The english directory is symlinked to another location, but the other languages are all right there.

So for example, the doc you are looking for in Korean is here: https://github.com/cncf/tag-app-delivery/blob/main/website/content/ko/wgs/operator/whitepaper/index.md

Hope that helps and happy to work with you on this 👍

**Comment by ramiyengar** [comment]
I created a PR to help resolve this issue.

**Comment by ramiyengar** [comment]
I will take a look at each of the languages and see where I need to update the link.

**Comment by naivary** [comment]
I'm sorry I could'nt do it in my free time. Got to do with uni currently :(
